src/_components/PlayerSection/PlayerSection.js

```diff
@@ -1,15 +1,45 @@
-import React from "react";
-import "./PlayerSection.scss";
+import { connect } from "react-redux";
+import React, { Component } from "react";
+import * as actions from "../../_redux/actions";
+import ActivatedCards from "../ActivatedCards/ActivatedCards";
 import PlayerDeck from "../PlayerDeck/PlayerDeck";
 import PlayerHand from "../PlayerHand/PlayerHand";
-import ActivatedCards from "../ActivatedCards/ActivatedCards";
 
-export default function PlayerSection() {
-  return (
-    <section className='PlayerSection'>
-      <ActivatedCards />
-      <PlayerDeck />
-      <PlayerHand />
-    </section>
-  );
+import "./PlayerSection.scss";
+
+export class PlayerSection extends Component {
+  componentDidMount = () => {
+    //fetch request for player state;
```

Comment:
@BrennanAyers @Patrick-Duvall  @DavisC0801  @KevinKra 

**This is where I'd use the endpoint to fetch player state** 

![](https://i.imgur.com/CD5SIsj.jpg)